.editorconfig

```diff
@@ -1,3 +1,5 @@
 [*.{ts,json,js}]
 indent_size = 2
 indent_style = space
+[*.snap]
+trim_trailing_whitespace = false
```

Comment:
Since markdown uses two spaces for new lines, this is required. Otherwise, `test.snap` files will trim trailing whitespaces whenever edited, resulting in incorrect snapshots and failed tests.